client/devdocs/design/blocks.jsx

```diff
@@ -115,6 +116,8 @@ export default class AppComponents extends React.Component {
 					filter={ this.state.filter }
 					section="blocks"
 				>
+					{ /* To Be reordered */
+					isEnabled( 'simple-site-rename/devdocs' ) && <SimpleSiteRenameForm /> }
```

Comment:
Looks like the `currentDomainName` prop was missed here..?

Reply:
Oops! I'd renamed the prop but not in the example 😬 thanks for catching that!